src/js/fluxcontroller.js

```diff
@@ -308,8 +308,13 @@ define(function (require, exports, module) {
                     return actionName[0] !== "_";
                 })
                 .forEach(function (actionName) {
-                    var action = actionModule[actionName],
-                        name = actionModuleName + "." + actionName,
+                    var action = actionModule[actionName];
```

Comment:
I exported more constants due to the refactoring. https://github.com/adobe-photoshop/spaces-design/pull/2225/files#diff-d036559733279e3c642c1fffe3d0fbc8R85